llvm/lib/Target/ARM/ARMFrameLowering.cpp

```diff
@@ -2190,9 +2213,12 @@ bool ARMFrameLowering::restoreCalleeSavedRegisters(
   auto IsDPRCS1 = [&CheckRegArea](unsigned Reg) {
     return CheckRegArea(Reg, SpillArea::DPRCS1);
   };
+  auto IsGPRCS3 = [&CheckRegArea](unsigned Reg) {
+    return CheckRegArea(Reg, SpillArea::GPRCS3);
+  };
 
   if (PushPopSplit == ARMSubtarget::SplitR11WindowsSEH) {
```

Comment:
Can this `if` be removed now that the blocks are always pushed in the same order?

Reply:
Right! Good idea.